cluster/ubuntu/util.sh

```diff
@@ -438,14 +438,19 @@ function kube-down {
 function prepare-push() {
   #Not yet support upgrading by using local binaries.
   if [[ $KUBE_VERSION == "" ]]; then
-     echo "Upgrading nodes to local binaries is not yet supported.Please specify the version"
-     exit 1
+    if [[ ! -d "ubuntu/binaries" ]]; then
```

Comment:
I know this style was not introduced in this change but this would be more readable if it used absolute paths from $KUBE_ROOT. I'm not sure what directory I'm in just by looking at this. I would need to understand the entire script to have enough context to make a change in this single spot. Is there a standard?